src/main/flight/pid.c

```diff
@@ -400,6 +464,7 @@ void pidUpdateTpaFactor(float throttle)
         pidRuntime.tpaFactorYaw = pidRuntime.tpaFactor;
         break;
     }
+    updateStermTpaFactors();
```

Comment:
a getter function should not set things.

Reply:
this is inside `pidUpdateTpaFactor`, no getters here

Reply:
Yes sorry looked like part of getTpaFactorClassic, missed one refresh